lib/sles4sap_publiccloud.pm

```diff
@@ -226,7 +226,10 @@ sub sles4sap_cleanup {
     type_string('', terminate_with => 'ETX');
 
     qesap_cluster_logs();
-    qesap_supportconfig_logs(provider => get_required_var('PUBLIC_CLOUD_PROVIDER'));
+    if (get_var('SUPPORTCONFIG') || ((defined $self->{result}) && ($self->{result} eq 'fail'))) {
+        record_info('Collect supportconfig logs');
```

Comment:
I think we do not need the record_info here. What kind of extra information does it provide other than "we was here"?